server/router.py

```diff
@@ -13,8 +13,23 @@
 
 @router.post("/challenge")
 async def challenge(challenge: Challenge):
-    await make_challenge([challenge.challenger, challenge.challenged])
-    return challenge
+    try:
+        await make_challenge(
+            challenge.challenger,
+            challenge.challenged,
+            challenge.tournament_id,
+            challenge.game_name
```

Comment:
The web passes the name of the game now?

Reply:
Not necessarily, tournament_id and game_name have a default value (see [models.py](102/files#diff-d97741605a183dfd2c9975818dd030d80edfef76b2dd3fa5f696604068146b8a)) and are optional parameters